options/ansi/include/stdio.h

```diff
@@ -51,6 +52,10 @@ struct __mlibc_file_base {
 
 	// EOF and error bits.
 	int __status_bits;
+
+	// PID of the popen'ed process
+	pid_t __popen_pid;
+	int __popen_pipe;
```

Comment:
This is an ABI break so it needs to be coordinated with the next mlibc major release.